researchapp/templates/consent.jinja2

```diff
@@ -6,20 +6,18 @@
 
 {% block body %}
 <div>
-    <form action="http://52.39.26.206:9000/authorize/" id="authorize-form">
-        <input type="hidden" name="client_id" value="{{ oauth_config.client_id }}" />
+    <form action="{{ provider.authorize_url }}" id="authorize-form">
+        <input type="hidden" name="client_id" value="app-demo" />
         <input type="hidden" name="response_type" value="code" />
         <input type="hidden" name="scope" value="patient/*.read offline_access" />
         <input type="hidden" name="redirect_uri" value="http://52.39.26.206:6543/authorized" />
         <input type="hidden" name="state" value="123" />
```

Comment:
A best practice is to ensure that `state` is never re-used -- and generally CSRF tokens are fixed for an entire user session (or even longer). So we should generate state separately from CSRF (e.g. could calculate a HMAC over a fixed CSRF token + an incrementing integer, and use that as state).